connector/protobuf/README.md

```diff
@@ -34,3 +34,17 @@ export SPARK_PROTOC_EXEC_PATH=/path-to-protoc-exe
 
 The user-defined `protoc` binary files can be produced in the user's compilation environment by source code compilation, 
 for compilation steps, please refer to [protobuf](https://github.com/protocolbuffers/protobuf).
+
+### Work with IDE
```

Comment:
1. I'm not sure whether this item should be added. 
2. On the other hand, there are many modules that use protobuf, such as `core` (in fact, the `StoreTypes `generated in the `core` module have exceeded the default limit of IntelliJ),  `connect-common`, so if we need to add this one, should put it in a more general place instead of in `protobuf` module.

Reply:
> On the other hand, there are many modules that use protobuf, such as core (in fact, the StoreTypes generated in the core module have exceeded the default limit of IntelliJ), connect-common, so if we need to add this one, should put it in a more general place instead of in protobuf module.

Yes, the reason why I want to add this because I find `StoreTypes` can't be indexed by default. At first I want add this in to core's README.md (but core doesn't have one), so I think add this in protobuf module not bad choice. If put this into core's README.md, I would create new README.md in core module and only describe some protobuf problems. it's weird.